doc/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<!-- This is needed to redirect to the main version when building docs -->
+<html>
+  <head>
+    <title>Redirecting to main branch</title>
+    <meta charset="utf-8">
+    <meta http-equiv="refresh" content="0; url=./main/index.html">
+    <link rel="canonical" href="https://parthenon-hpc-lab.github.io/parthenon/jmm/sphinx-doc-skeleton/index.html">
```

Comment:
Change right before merge:
```suggestion
   <link rel="canonical" href="https://parthenon-hpc-lab.github.io/parthenon/develop/index.html">
```

Reply:
Probably also in the line above, doesn't it?